official/landing_pages/carrier_language_api/buy-label.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using EasyPost;
+using EasyPost.Models.API;
+using EasyPost.Parameters;
+
+namespace EasyPostExamples
+{
+    public class Examples
+    {
+        public static async Task Main()
+        {
+            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;
+
+            var client = new EasyPost.Client(apiKey);
+
+            Parcel parcel = await client.Parcel.Create(new Dictionary<string, object>() {
+                { "predefined_package", "Parcel" },
+                { "weight", 28 }
+            });
```

Comment:
We want to guide people to use the new Parameters objects, not dictionaries.

```suggestion
Parcel parcel = await client.Parcel.Create(
  new Parameters.Parcel.Create {
    PredefinedPackage = "Parcel",
    Weight = 28
  }
);
```